requirements.txt

```diff
@@ -0,0 +1,2 @@
+paramiko
```

Comment:
It's a good idea to lock the dependency to a known working version so your package doesn't break when one dependency gets updated on a backward incompatible way